core/templates/dev/head/pages/landing/stewards/landing_page_stewards.html

```diff
@@ -26,7 +26,7 @@ <h2 class="oppia-stewards-landing-h2" style="padding-right: 15px;">Support from
             </div>
           </div>
         </div>
-        <div style="position: absolute; top: 50px">
+        <div class="oppia-stewards-landing-background-icon-row-container" style="position: absolute; top: 50px;;">
```

Comment:
Remove the double semi-colon.

Also, could you tell what the class is for ? (I think we should change the name!)

Reply:
I made that class to avoid inline styling in the div since inline styling is [discouraged](https://github.com/oppia/oppia/pull/6187#discussion_r254018094). 
The div contains the three landing background icon rows and hence the name. What another name do you suggest?

Reply:
Also, I needed to style the div for the specified screen size and hence the need for a class name!

Reply:
Thanks for the explanation. Though, I didn't had questions as to why the class is created but the name of class, and its fine!

Could you also move the other two style attrs in the class then ?

Reply:
Done! PTAL!